scripts/pyro_camera_server.py

```diff
@@ -5,11 +5,19 @@
 pyro_camera_local.yaml config file. The camera servers should be started after the name server,
 but before POCS.
 """
+import os, sys
 import argparse
-
 from huntsman.utils.pyro import run_camera_server
 
 if __name__ == "__main__":
+    
+    #Mount the SSHFS 
+    sys.path.append(
+            os.path.join(os.path.expandvars('$HUNTSMAN_POCS'), 'scripts'))
+    from auto_sshfs_mount import MountSSHFS
+    MountSSHFS()
```

Comment:
This probably should go in the `run_camera_server` function instead of this script: https://github.com/AstroHuntsman/huntsman-pocs/blob/develop/huntsman/utils/pyro/__init__.py#L114

That way the automounting will still happen if the camera server is started by calling the function directly instead of this script, and you can add attempted unmounting to the clean up code that gets executed when the camera server is stopped (here: https://github.com/AstroHuntsman/huntsman-pocs/blob/develop/huntsman/utils/pyro/__init__.py#L150)